app/controllers/tag_groups_controller.rb

```diff
@@ -0,0 +1,58 @@
+class TagGroupsController < ApplicationController
+  skip_after_filter :verify_policy_scoped, only: :index
```

Comment:
I think this method was deprecated in Rails 2.3.8+, and we're using 4.2.7.1 https://apidock.com/rails/ActionController/Filters/ClassMethods/skip_after_filter